File: data/python/MINIMUM_SUM_TWO_NUMBERS_FORMED_DIGITS_ARRAY_TRANSLATED.java
====================
public static int f_gold ( int [ ] arr , int n ) {
  Arrays . sort ( arr ) ;
  int a = 0 ;
  int b = 0 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    if ( ( i % 2 != 0 ) && ( arr [ i ] > 0 ) ) a = a * 10 + arr [ i ] ;
    else b = b * 10 + arr [ i ] ;
  }
  return a + b ;
}

File: data/java/MINIMUM_SUM_TWO_NUMBERS_FORMED_DIGITS_ARRAY_TRANSLATED.py
====================
def f_gold ( arr , n ) :
    arr = np.asarray ( arr )
    a , b = 0 , 0
    for i in range ( n ) :
        if i % 2 != 0 :
            a = a * 10 + arr [ i ]
        else :
            b = b * 10 + arr [ i ]
    return a , b

